blueprints/dumb/files/__root__/__dumb__/__name__.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component, PropTypes } from 'react'
```

Comment:
Starting with react 15.5.5 `PropTypes` are now separate module, this code will issue a warning.
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes